tensorflow/core/util/util.h

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 
 #include "tensorflow/core/framework/tensor_shape.h"
 #include "tensorflow/core/lib/core/stringpiece.h"
+#include "tensorflow/core/util/onednn_env_vars.h"
 
 namespace tensorflow {
```

Comment:
Please remove the declaration of `IsMKLEnabled()` in this file (lines 60-61).

Reply:
Thanks for your comment, removed it as per request